Documentation/adding-new-repository.md

```diff
@@ -20,6 +20,7 @@ Create an empty repository in GitHub and push the OS version of it. Notable chan
 2. Update the `.gitlab-cy.yml` file to set the `FIRST_ENT_COMMIT` in the `test:check-license` job to the commit sha from point 1 (see [example](https://github.com/mendersoftware/deployments-enterprise/blob/master/.gitlab-ci.yml#L35))
 3. Modify references in README
 4. Modify variables in pipeline
+5. Create the `mendersoftware/name-of-service` ECR repository in the AWS Account with ID `175683096866` to be served by `registry.mender.io`
```

Comment:
This doesn't belong here. See [Docker image](https://github.com/mendersoftware/mender-qa/blob/9e99bf42a7156389cd6332e162ebebf081902268/Documentation/adding-new-repository.md#docker-images) section. We can extend the text there with the more complete text we have here.

Additionally, we can add a note in this part saying "Create repository, see Docker image(s))

Reply:
wow. it is there:
`* For closed source repository, create a new private repository in registry.mender.io`